code/modules/food/food/condiment.dm

```diff
@@ -441,30 +453,87 @@
 
 //End of MRE stuff.
 
-/obj/item/weapon/reagent_containers/food/condiment/flour
-	name = "flour sack"
-	desc = "A big bag of flour. Good for baking!"
+/obj/item/weapon/reagent_containers/food/condiment/advanced/flour
```

Comment:
Why advanced? Is this already a class that does things?

Reply:
This is to add functionality for updating the sprite as it gets emptier without making this the case for every condiment bottle out there.